Resources/Prototypes/_Goobstation/Recipes/Reactions/fun.yml

```
@@ -109,4 +112,72 @@
+- type: reaction
+  id: BirdFlu
+  reactants:
+    Egg:
+      amount: 6
+    Impedrezene:
+      amount: 6
+    Ephedrine:
+      amount: 6
+  products:
+    BirdFlu: 20
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)